lib/firehose/rack/consumer.rb

```diff
@@ -78,6 +78,9 @@ def self.post_subscriptions(request)
         end
 
         subs
+      rescue JSON::ParserError
+        Firehose.logger.warn "Consumer.post_subscriptions: JSON::ParserError for request body: #{body.inspect}"
```

Comment:
might be good to add a template method here so you can hook in handling to get e.g. the backtrace

Reply:
That's a good idea. I'll put up a PR for it soon.